assets/airports/eidw.json

```diff
@@ -1,1329 +1,5087 @@
+          ["N53.304294300", "W6.474616500"],
+          ["N53.305128200", "W6.474205700"],
+          ["N53.305948300", "W6.473722600"],
+          ["N53.306752200", "W6.473168500"],
+          ["N53.307537800", "W6.472544900"],
+          ["N53.308303100", "W6.471853500"],
+          ["N53.309045700", "W6.471096000"],
+          ["N53.309763700", "W6.470274600"],
+          ["N53.310455300", "W6.469391300"],
+          ["N53.311118400", "W6.468448600"],
+          ["N53.311751100", "W6.467448900"],
+          ["N53.312351800", "W6.466395100"],
+          ["N53.312918800", "W6.465289900"],
+          ["N53.313450400", "W6.464136300"],
+          ["N53.313945300", "W6.462937400"],
+          ["N53.314402000", "W6.461696600"],
+          ["N53.314819100", "W6.460417300"],
+          ["N53.315195600", "W6.459102900"],
+          ["N53.315530400", "W6.457757100"],
+          ["N53.315822500", "W6.456383600"],
+          ["N53.316071100", "W6.454986100"],
+          ["N53.316275400", "W6.453568700"],
+          ["N53.316435000", "W6.452135100"],
+          ["N53.316549300", "W6.450689400"],
+          ["N53.316618000", "W6.449235600"],
+          ["N53.316640900", "W6.447777800"]
+        ]
+      }
+  ],
+  "runways":[
+    {
+      "name": ["10", "28"],
+      "end": [["N53d25m20.75", "W6d17m24.27", "242ft"], ["N53d25m12.94", "W6d15m02.08", "202ft"]],
+      "ils": [true, true]
     },
-    "icao": "EIDW",
-    "iata": "DUB",
-    "magnetic_north": -4,
-    "ctr_radius": 80,
-    "ctr_ceiling": 24500,
-    "initial_alt": 9000,
-    "position": ["N53d25m17.00", "W6d16m12.00", "242ft"],
-    "rr_radius_nm": 5.0,
-    "rr_center": ["N53d25m17.00", "W6d16m12.00"],
-    "has_terrain": false,
-    "wind": {
-        "angle": 240,
-        "speed": 15
+    {
+      "name": ["16", "34"],
+      "end": [["N53d26m13.16", "W6d15m43.12", "217ft"], ["N53d25m11.66", "W6d14m58.54", "202ft"]],
+      "ils": [true, true]
+    }
+  ],
+  "sids": {
+    "SUROX": {
+      "icao": "SUROX",
+      "name": "Surox",
+      "suffix": {"10":"6E", "28":"1A", "16":"5M", "34":"1G"},
+      "rwy": {
+        "10": [["DW551", "A30+"], "LEMTA", "DODIG"],
+        "28": [["OE", "A30+"], "_LIFFY1", "_LIFFY2", "_LIFFY3", "_LIFFY4", "_LIFFY5", "_LIFFY6", "_LIFFY7", "_LIFFY8", "_LIFFY9", "_LIFFY10", "DWW04"],
+        "16": [["DWS77", "A30+"], "_BATED1", "_BATED2", "_BATED3", "_BATED4", "_BATED5", "_BATED6", "_BATED7", "_BATED8", "_BATED9", "_BATED10", "_BATED11", "_BATED12", "_BATED13", "_BATED14", "_BATED15", "_BATED16", "_BATED17", "_BATED18", "_BATED19", "_BATED20", "_BATED21", "_BATED22", "_BATED23", "_BATED24", "_BATED25", "_BATED26", "_BATED27", "_BATED28", "_BATED29", "BATED", "ERUDA", "DODIG"],
+        "34": [["DWN26", "A30+"], "_BAL1", "_BAL2", "_BAL3", "_BAL4", "_BAL5", "_BAL6", "_BAL7", "_BAL8", "_BAL9", "_BAL10", "_BAL11", "DWW06", "DWW04"]
+      },
+      "body": ["SUROX"],
+      "exitPoints": {
+        "ENULA": ["ENULA"]
+      },
```

Comment:
![image](https://cloud.githubusercontent.com/assets/5103735/25785484/9d12e252-334f-11e7-9cb0-d0489ea8624a.png)

The procedure ends at `SUROX`, so `SUROX` should be the only exitPoint. To get aircraft to follow the SUROX departure to SUROX then ENULA, that can be added to the route in the spawnPattern. But from what my charts show me, it shouldn't be a part of the departure procedure itself.

Similar cases in other SIDs.